Doc/library/tkinter.rst

```diff
@@ -67,13 +67,33 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1, sync=0, use=None)
+
+   Return a toplevel Tk widget, which is usually the main window of an
+   application.  Each instance has its own associated Tcl interpreter.
+
+   The :class:`Tk` class is typically instantiated using all default values.
+   However, the following keyword arguments are currently recognized:
+
+   *screenName*
+      Sets the DISPLAY environmental variable. (X11 only)
+   *baseName*
+      Reads the profile file :file:`baseName.tcl` into the Tcl interpreter and
+      calls ``exec`` on the contents of :file:`baseName.py`.
+   *className*
+      Name of the widget class.  Used as a profile file name
+      (:file:`className.tcl` and :file:`className.py` load before the
+      *baseName* files), and also used as the name with which Tcl is
+      invoked (*argv0* in *interp*).
+   *useTk*
+      If True, initialize the Tk subsystem.  :func:`Tcl` sets this to False.
```

Comment:
```rst
``False``
```

Reply:
I either do not understand or do not see the relevance of the second sentence.
Edit: I presume the markup will cause this to be linked to the Tcl function below, which I did not initially think of.  I suggest expanding to (markup missing) "The Tcl function ... ."  I normally interpret 'Tcl' to mean the Tcl languate or interpreter, as I would with 'Python'.  Normal readers will not see the tag, nor know of the function.